OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -1,42 +1,51 @@
 Apache HTTP Server with Oracle WebLogic Server Proxy Plugin on Docker
 ===============
-This project includes a quick start Dockerfile and samples for standalone Apache HTTP Server with the 12.2.1.3.0 Oracle WebLogic Server Proxy Plugin based on Oracle Linux. The certification of Apache on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize, tweak, or create from scratch, new scripts and Dockerfiles.
+This project includes a Dockerfile and samples for standalone Apache HTTP Server with the 12.2.1.3.0 and 12.2.1.4.0 Oracle WebLogic Server Proxy Plugin based on Oracle Linux. This is a generic Docker File and can be used to build apache images of different versions and to support containers to run on differnet ports i.e 80 and 8080. The certification of Apache on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize, tweak, or create from scratch, new scripts and Dockerfiles.The only supported versions are 12.2.1.3.0 & 12.2.1.4.0.
```

Comment:
Docker File -> Dockerfile
apache -> Apache
differnet ports i.e 80 and 8080.->different ports, such as, 80 and 8080.
any file presented in this repository -> any file in this repository 
& -> and